lua/telescope/actions.lua

```diff
@@ -12,6 +12,15 @@ local actions = setmetatable({}, {
   end
 })
 
+local action_mt = {
```

Comment:
We need to make the accessable for the user so that he can use it when defining his own actions. This is necessary if he wants to add his actions with actions.goto_file_selection_edit, etc.